mon-pix/app/services/pix-metrics.js

```diff
@@ -4,9 +4,9 @@ export default class PixMetricsService extends Service {
   @service metrics;
   @service router;
 
-  trackEvent({ event: _, 'pix-event-name': eventName, ...props }) {
+  trackEvent(eventName, { event: _, ...props }) {
```

Comment:
La propriété `event` ne sert à rien et les props ne sont pas obligatoire, donc mettre une valeur par défaut.
```suggestion
  trackEvent(eventName, props = {}) {
```